resources/views/components/game/top-achievers/mastery-row.blade.php

```diff
@@ -0,0 +1,20 @@
+@props([
+    'rank' => 1,
+    'masteryUser' => null, // User
+    'masteryDate' => null, // Carbon
+    'iconSize' => 'xs',
+])
+
+@if ($masteryUser->id === request()->user()->id)
+    <tr style='outline: thin solid'>
+@else
+    <tr>
+@endif
+
+        <td class="text-right">{{ $rank }}</td>
+        <td class="whitespace-nowrap">
+            <x-user.avatar display="icon" :icon-size="$iconSize" :user="$masteryUser" />
+            <x-user.avatar hasHref="true" :user="$masteryUser" />
```

Comment:
Tooltips don't seem to be working for these avatar components.

Reply:
It looks like the `<x-avatar>` component isn't fully fleshed out yet: https://github.com/RetroAchievements/RAWeb/blob/c5d15e85961be6bbcb06a5d35a9cb08f5e62d3ce/resources/views/components/avatar.blade.php#L22-L34

Replaced with `userAvatar` function call.